csrankings-g.csv

```diff
@@ -644,7 +644,7 @@ Gregoris Mentzas,NTUA,http://imu.ntua.gr/users/gmentzas,WdybrQYAAAAJ
 Gregory B. White,University of Texas at San Antonio,http://www.cs.utsa.edu/facstaff,yT6jxa8AAAAJ
 Gregory Bodwin,University of Michigan,https://bodwin.engin.umich.edu,NWpUSd4AAAAJ
 Gregory Butler,Concordia University,http://www.cs.concordia.ca/~gregb,K2vSwvQAAAAJ
-Gregory D. Abowd,Georgia Institute of Technology,http://ubicomp.cc.gatech.edu/gregory-d-abowd,gb8sbdcAAAAJ
+Gregory D. Abowd,Northeastern University,https://coe.northeastern.edu/people/abowd-gregory/,gb8sbdcAAAAJ

```

Comment:
Not in CS. See https://www.khoury.northeastern.edu/about/people/. Please remove.

Reply:
Have removed Gregory as he is now not working at Gatech. As well as he is not working in CS Department, so I remove Gregory from the list.